docs/source/frida_bindings.rst

```diff
@@ -67,6 +67,98 @@ Analysis
     :members: QBDI.prototype.getInstAnalysis, QBDI.prototype.getInstMemoryAccess, QBDI.prototype.getBBMemoryAccess
     :member-order: alphabetical
 
+.. js:class:: InstAnalysis
+
+  Object describes the Analysis of the current instruction
```

Comment:
**This** object  or Object that describes ?

Reply:
fix